framework/UPGRADE.md

```diff
@@ -57,6 +57,12 @@ Upgrade from Yii 2.0.x
   providing ability to setup custom mail headers. Make sure your provide implementation for those methods, while
   creating your own mailer solution.
 
+* During mail view rendering `$message` variable is no longer set by default to be an instance of `yii\mail\MessageInterface`,
```

Comment:
There is a related issue #4147 , which is closed.
At 2.0.x there is a problem in views params conflict: if `$params`does NOT hold `message` key it will be set by message instance - otherwise message instance is unavailable during rendering.

Placing `message` to the view context ensures it is always available and does not conflict with any view param.